src/helpers.test.js

```diff
@@ -375,6 +378,45 @@ describe('getPrettierEslintOptions', () => {
   });
 });
 
+describe('getEditorConfigOptions', () => {
+  test('parse editorconfig by filename', () => {
+    getEditorConfigOptions('filename');
+    expect(editorconfig.parseSync).toHaveBeenCalledWith('filename');
+  });
```

Comment:
Since we mock the implementation below, I think the above test can be safely eliminated without losing any error checking. We could also just put this expectation in the below test in addition to the other expectation as well.